### PR TITLE
EI-1134 Create SPARQLUpdate wrapper in the gRPC client

### PR DESCRIPTION
**Pull request by zoltaniotic**

### CODE REVIEW

File: src/sparqlUpdate.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                return;
+            }
+            if (responseMessage == null) {
+                const msg = "sparqlUpdateApi: Response message is null.";
+                // eslint-disable-next-line no-console
+                console.warn(msg);
+                reject(new Error(msg));
+                return;
+            }
+            const responsePayload = responseMessage;
```
Comment (bkr-iotic) [inline src/sparqlUpdate.ts:70]: What's the point of this rename?

File: src/sparqlUpdate.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { grpc } from "@improbable-eng/grpc-web";
+import { Timestamp } from "google-protobuf/google/protobuf/timestamp_pb";
+import * as pbMetaService from "./client/iotics/api/meta_pb_service";
+import { eventChannel } from "redux-saga";
```
Comment (bkr-iotic) [inline src/sparqlUpdate.ts:22]: unused? linter should get this

**Review by bkr-iotic — COMMENTED** [review]
nice! 2 minor things

File: src/sparqlUpdate.ts (hunk 1 above)
Comment (zoltaniotic) [reply src/sparqlUpdate.ts]: Nothing at all, leftover. Thanks. Removed

**Review by bkr-iotic — APPROVED** [review]
nice!

**Pull request merged**